### ISSUE TITLE
Bug: Redunant private keyword being added

### ISSUE THREAD

**Issue by Grauenwolf** [codecadwallader/codemaid#458]
### Environment
- Visual Studio 2017
- CodeMaid version: 10.4
- Code language: C#

### Description
When running the cleanup command, Codemaid is adding the `private` keyword where it isn't needed. For example, 

    readonly TimeSpan m_PollingInterval;

becomes

    private readonly TimeSpan m_PollingInterval;

### Steps to recreate
Run the cleanup command using ctrl-m, spacebar

### Current behavior
Adding redundant access modifiers.

### Expected behavior
Remove redundant access modifiers. (Or at least stop adding them.)

**Comment by codecadwallader** [comment]
Thanks for reporting the issue.  By default we add explicit access modifiers to avoid confusion.  You can disable this at CodeMaid->Options->Cleaning->Insert->Insert explicit access modifiers and toggle any/all member types you do not want to be affected.

Hope it helps!

**Comment by Grauenwolf** [comment]
Then this should be marked as a documentation bug. Maybe add some text or
a tool tip on that settings page to make it clearer that's what it does.


--
Jonathan Allen
Scholars of Alcala <https://www.meetup.com/Scholars-of-Alcala>
619-933-8527 <#>

On Thu, Jun 15, 2017 at 3:15 AM, Steve Cadwallader <notifications@github.com

**Comment by codecadwallader** [comment]
I believe this is already covered at http://www.codemaid.net/documentation/#cleaning

**Comment by Grauenwolf** [comment]
Right, but there's no clear path from the options page to the documentation. The options page needs a Help button.
